site/src/components/home/explore/index.js

```diff
@@ -104,6 +121,31 @@ export default function Explore() {
     setDropdownVisible(false);
   }
 
+  function onInputKeyDown(e) {
+    if (!dropdownVisible) {
+      return;
+    }
+
+    const { code } = e;
+
+    if (code === "Enter") {
+      handleExplore();
+    } else if (code === "ArrowUp") {
+      if (selectedIndex > 0) {
+        setSelectedIndex(selectedIndex - 1);
+      }
+    } else if (code === "ArrowDown") {
+      if (selectedIndex < hints.length - 1) {
+        setSelectedIndex(selectedIndex + 1);
+      }
+    }
+  }
+
```

Comment:
What about adding `code === "Escape"` ?

Reply:
browser native supported: will blur the input